src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRentalPrice = 40;
+  const discountForSevenDays = 50;
+  const discountForThreeDays = 20;
+  const rentalDays = days;
+  const totalCost = dailyRentalPrice * rentalDays;
+  let finalCost;
+
+  if (rentalDays < 3) {
+    finalCost = totalCost;
+  } else if (rentalDays >= 3 && rentalDays < 7) {
```

Comment:
You can simplify the condition by removing 'rentalDays >= 3' because it's already checked in the previous 'if' statement. This aligns with checklist item 4.